### PR TITLE
Adding "Two-Stream Network for Sign Language Recognition and Translation"

### PR DESCRIPTION
**Pull request by cleong110**
Paper link: https://papers.nips.cc/paper_files/paper/2022/hash/6cd3ac24cdb789beeaa9f7145670fcae-Abstract-Conference.html

My notes on this, including initial summary, ChatGPT rewrite suggestions, etc: https://github.com/cleong110/sign-language-processing.github.io/issues/9.

Should all conform to style guide, let me know if I missed aught.

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -207,6 +207,18 @@ Linear gloss annotations have been criticized for their imprecise representation
 These annotations fail to capture all the information expressed simultaneously through different cues, 
 such as body posture, eye gaze, or spatial relations, leading to a loss of information that can significantly affect downstream performance on SLP tasks [@yin-read-2020-better;@muller-etal-2023-considerations].
 
+@muller-etal-2023-considerations conduct an extensive review of the use of glosses in sign language translation research and make the following recommendations for research using glosses:
+* Demonstrate awareness of limitations of gloss approaches and explicitly discuss them.
+* Focus on datasets beyond RWTH-PHOENIX-Weather-2014T [@cihan2018neural]. 
+Openly discuss the limited size and linguistic domain of this dataset.
+* Use metrics that are well-established in MT.
+If BLEU is used, compute it with SacreBLEU,
```
Comment (AmitMY) [inline src/index.md:215]: cite BLEU here, cite SacreBLEU (because we are talking about them, not in passing)

File: src/index.md [hunk 2]
```diff
@@ -207,6 +207,18 @@ Linear gloss annotations have been criticized for their imprecise representation
...
+report metric signatures and disable internal
+tokenization for gloss outputs.
```
Comment (AmitMY) [inline src/index.md:217]: make the three above rows, one line. guideline is to only break lines on periods, to make full sentences

File: src/index.md [hunk 3]
```diff
@@ -207,6 +207,18 @@ Linear gloss annotations have been criticized for their imprecise representation
...
+* Demonstrate awareness of limitations of gloss approaches and explicitly discuss them.
+* Focus on datasets beyond RWTH-PHOENIX-Weather-2014T [@cihan2018neural]. 
+Openly discuss the limited size and linguistic domain of this dataset.
+* Use metrics that are well-established in MT.
+If BLEU is used, compute it with SacreBLEU,
+report metric signatures and disable internal
+tokenization for gloss outputs. 
+Do not compare to scores produced with a different or unknown evaluation procedure.
+* Given that glossing is corpus-specific, process glosses in a corpus-specific way, informed by transcription conventions.
+* Optimize gloss translation baselines with methods shown to be effective for low-resource MT.
```
Comment (AmitMY) [inline src/index.md:220]: I don't expect that you expect it, but it looks like this: (https://dillinger.io/)
image

File: src/index.md [hunk 4]
```diff
@@ -678,8 +690,11 @@ When pretraining, all augmentations show improvements over the baseline for RWTH
 #### Text-to-Gloss
 Text-to-gloss, an instantiation of sign language translation, is the task of translating between a spoken language text and sign language glosses.
 It is an appealing area of research because of its simplicity for integrating in existing NMT pipelines, 
-despite recent works such as @yin-read-2020-better and@muller2022considerations claim that glosses are an inefficient representation of sign language,
+despite recent works such as @yin-read-2020-better and @muller-etal-2023-considerations claim that glosses are an inefficient representation of sign language,
```
Comment (AmitMY) [inline src/index.md:693]: good catch!

File: src/index.md [hunk 5]
```diff
@@ -721,6 +736,10 @@ and so they have broken the dependency upon costly annotated gloss information i
...
 
+<!-- Really should put MMTLB here, a number of papers cite it including chen2022, which actually builds on it directly, cites it as a source for "mBART is good for SLT", etc. -->
+
+@chen2022TwoStreamNetworkSign present a two-stream network for sign language recognition (SLR) and translation (SLT), utilizing a dual visual encoder architecture to encode RGB video frames and pose keypoints in separate streams. These streams interact via bidirectional lateral connections. For SLT the visual encoders, based on an S3D backbone [@xie2018SpatiotemporalS3D], output to a multilingual translation network built on mBART[@liu-etal-2020-multilingual-denoising]. The model achieves state-of-the-art performance on the RWTH-PHOENIX-Weather-2014 [@dataset:forster2014extensions], RWTH-PHOENIX-Weather-2014T [@cihan2018neural] and CSL-Daily [@dataset:huang2018video] datasets.
```
Comment (AmitMY) [inline src/index.md:741]: break into lines.
space after `mBART` before citation 

Great writing otherwise!

**Review by AmitMY — CHANGES_REQUESTED** [review]

File: src/index.md (hunk 3 above)
Comment (cleong110) [reply src/index.md:220]: Oh that's helpful! I can preview it in VS Code as well, but this tool makes it really easy to check small sections

File: src/index.md (hunk 4 above)
Comment (cleong110) [reply src/index.md:691]: Thanks! :)

**Comment by cleong110** [comment]
Working on the merge conflict now. It seems I need to: 
1. Sync fork on Github so my version of master is updated
2. `git pull` on my local master
3. `git merge master` in my local version of cleong110:paper/chen2022TwoStreamNetworkSign

File: src/index.md [hunk 6]
```diff
@@ -722,6 +735,13 @@ and so they have broken the dependency upon costly annotated gloss information i
 
 @shi-etal-2022-open introduce OpenASL, a large-scale American Sign Language (ASL) - English dataset collected from online video sites (e.g., YouTube), and then propose a set of techniques including sign search as a pretext task for pre-training and fusion of mouthing and handshape features to improve translation quality in the absence of glosses and in the presence of visually challenging data.
 
+<!-- Really should put MMTLB here, a number of papers cite it including chen2022, which actually builds on it directly, cites it as a source for "mBART is good for SLT", etc. -->
+
+@chen2022TwoStreamNetworkSign present a two-stream network for sign language recognition (SLR) and translation (SLT), utilizing a dual visual encoder architecture to encode RGB video frames and pose keypoints in separate streams. 
+These streams interact via bidirectional lateral connections. 
+For SLT the visual encoders, based on an S3D backbone [@xie2018SpatiotemporalS3D], output to a multilingual translation network built on mBART [@liu-etal-2020-multilingual-denoising].
```
Comment (AmitMY) [inline src/index.md:742]: This fractured sentence is strange.
Moving a comma about:
> For SLT, the visual encoders based on an S3D backbone [@xie2018SpatiotemporalS3D] output to a multilingual

Comment (AmitMY) [inline src/index.md:742]: `build on` - i think should be `built using` or just `using`

**Review by AmitMY — CHANGES_REQUESTED** [review]
minor corrections

File: src/index.md (hunk 6 above)
Comment (cleong110) [reply src/index.md:742]: Yeah that's better

**Pull request merged**